dvc/remote/base.py

```diff
@@ -156,6 +172,12 @@ def group(self, name):
     def cache(self):
         return getattr(self.repo.cache, self.scheme)
 
+    def checksum_types(self):
+        return self._checksum_types
+
+    def checksum_type(self):
+        return self._checksum_types[0]
+
```

Comment:
Make this properties as above?

Reply:
Also is that an error if we have more than one here? If yes than I suggest adding an assert.